lib/core/src/Cardano/Wallet/DB/Sqlite/Types.hs

```diff
@@ -765,10 +765,14 @@ instance PersistField POSIXTime where
         . posixSecondsToUTCTime
     fromPersistValue (PersistText time) =
         utcTimeToPOSIXSeconds <$>
-            parseTimeM True defaultTimeLocale
+            parseTimeM False defaultTimeLocale
```

Comment:
Don't recall why I did this. 🤔  Should try reverting.

Reply:
Probably either value would work fine, but if in doubt, don't make parsing more strict.